src/middleware/optionsMiddleware.ts

```diff
@@ -0,0 +1,15 @@
+import { CACHE_HEADERS } from '../constants/cache';
+import { Middleware } from './middleware';
+
+export class OptionsMiddleware implements Middleware {
```

Comment:
IMO each middleware should have documentation (JSDoc) above the class definition explaining what it does